tools/cli/internal/openapi/oasdiff_test.go

```diff
@@ -305,6 +335,13 @@ func TestOasDiff_mergeTags(t *testing.T) {
 				external: tc.inputExternal,
 			}
 			tc.wantErr(t, o.mergeTags())
+			if tc.expectedTags != nil {
+				assert.Equal(t, len(o.base.Spec.Tags), len(tc.expectedTags))
+				// check tag names are the same
+				for i := range o.base.Spec.Tags {
+					assert.Equal(t, o.base.Spec.Tags[i].Name, tc.expectedTags[i].Name)
+				}
```

Comment:
wouldn't the following do the same? it should account for order
```suggestion
				assert.Equal(t, tc.expectedTags, o.base.Spec.Tags))
```

Reply:
no it doesn't work as it is a list of objects, it complains about the addresses